### PR TITLE
Recognition improvement for Lonsonho X702A

### PR DESCRIPTION
**Pull request by marcinsu**
Hello
I added custom converter to add the option of LED indicator. Everything works fine the same as the X702A version (similar to https://github.com/Koenkk/zigbee-herdsman-converters/pull/6117 )
```
const {actionEnumLookup,commandsOnOff,deviceEndpointsidentify,onOff} = require('zigbee-herdsman-converters/lib/modernExtend');
const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
const tz = require('zigbee-herdsman-converters/converters/toZigbee');
const exposes = require('zigbee-herdsman-converters/lib/exposes');
const reporting = require('zigbee-herdsman-converters/lib/reporting');
const ota = require('zigbee-herdsman-converters/lib/ota');
const utils = require('zigbee-herdsman-converters/lib/utils');
const globalStore = require('zigbee-herdsman-converters/lib/store');
const e = exposes.presets;
const ea = exposes.access;
const tuya = require('zigbee-herdsman-converters/lib/tuya');


const definition = {
        fingerprint: tuya.fingerprint('TS0001', ['_TZ3000_t3s9qmmg']),
        model: 'X701A',
        vendor: 'Lonsonho',
        description: '1 gang switch with backlight',
        extend: [tuya.modernExtend.tuyaOnOff({indicatorMode: true})],
        configure: async (device, coordinatorEndpoint) => {
            await tuya.configureMagicPacket(device, coordinatorEndpoint);
            await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
        },
        whiteLabel: [
            tuya.whitelabel('Lonsonho', 'X701A', '1 gang switch with backlight', ['_TZ3000_t3s9qmmg'])
        ]
};

module.exports = definition;
```

It is looks the same as https://www.zigbee2mqtt.io/devices/X711A.html

image

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -3653,6 +3653,20 @@ const definitions: DefinitionWithExtend[] = [
             await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
         },
     },
+    {
+        zigbeeModel: ['TS0001'],
```
Comment (Koenkk) [inline src/devices/tuya.ts:3657]:
```suggestion
        fingerprint: tuya.fingerprint('TS0001', ['_TZ3000_t3s9qmmg']),
```

File: src/devices/tuya.ts [hunk 2]
```diff
@@ -3653,6 +3653,20 @@ const definitions: DefinitionWithExtend[] = [
...
+        model: 'X701A',
+        vendor: 'Lonsonho',
+        description: '1 gang switch with backlight',
+        extend: [tuya.modernExtend.tuyaOnOff({indicatorMode: true})],
+        whiteLabel: [
+            tuya.whitelabel('Lonsonho', 'X701A', '1 gang switch with backlight', ['_TZ3000_t3s9qmmg']),
```
Comment (Koenkk) [inline src/devices/tuya.ts:3663]: This can be removed

File: src/devices/tuya.ts [hunk 3]
```diff
@@ -3653,6 +3653,20 @@ const definitions: DefinitionWithExtend[] = [
             await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
         },
     },
+    {
+        zigbeeModel: ['TS0001'],
+        model: 'X701A',
+        vendor: 'Lonsonho',
```
Comment (Koenkk) [inline src/devices/tuya.ts:3659]: Please modify `lonsonho.ts` instead of `tuya.ts`

Comment (marcinsu) [reply src/devices/tuya.ts]: Ok, moved to `lonsonho.ts`.

**Comment by Koenkk** [comment]
Thanks!

Could you also submit a picture for the docs? This can be done by:
1. Create a fork by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/fork)
2. Go to the `public/images/devices` directory, *Add file* -> *Upload files*
3. Upload the files and press *Commit changes*
4. Press *Contribute* -> *Open pull request* -> update title/description -> *Create pull request*

Make sure that:
- The filename is `X701A.png`
- The size is 512x512
- The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))

**Comment by Koenkk** [comment]
Thanks!

**Comment by marcinsu** [comment]
Done: https://github.com/Koenkk/zigbee2mqtt.io/pull/3281

Thanks for help with PR changes :)

**Pull request merged**